src/main/java/org/lightcouch/View.java

```diff
@@ -583,7 +603,22 @@ public View tempView(MapReduce mapReduce) {
     }
 
     private String getKeyAsJson(Object... key) {
-        return (key.length == 1) ? gson.toJson(key[0]) : gson.toJson(key); // single or complex key
```

Comment:
Can we get rid of this method and use `getKeyAsJsonElement()` (and appropriate `toJson` calls) everywhere?
Otherwise we appear to have two very similar methods, but only one that handles the array cases correctly.